src/Sylius/Component/Core/OrderProcessing/OrderShipmentProcessor.php

```diff
@@ -75,7 +76,7 @@ public function process(BaseOrderInterface $order)
     }
 
     /**
-     * @param BaseOrderInterface $order
+     * @param BaseOrderInterface | OrderInterface $order
```

Comment:
We have an Assert [at the beginning of process](https://github.com/Sylius/Sylius/pull/8062/files/4caa5f5f99e9fbecefaec5b1501a9bcc9a7ac24d#diff-f20198fd413e52074801c922434669d5R57) so I am ok with an `OrderInterface` typehint in private methods